### PR TITLE
Update /desktop/developer copy for 16.10

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Updated copy from Google Doc

## QA

- Run code
- Cross ref with copy doc

## Issue / Card

Copy doc: https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit#
Trello: https://trello.com/c/dBAQM9zF

### CODE REVIEW

**Comment by barrymcgee** [comment]
[/desktop/developers update what's new](https://trello.com/c/dBAQM9zF/1872-desktop-developers-update-what-s-new)

File: templates/desktop/developers.html [hunk 1]
```diff
@@ -70,7 +70,26 @@
                 <li class="last-item">Python 3.5, Golang 1.6, Unity 7.3.2, Gtk 3.16.7, Compiz 0.9.12.1 and qt 5.4.2 updates</li>
             </ul>
         </div>
-        <p class="twelve-col"><a class="external" href="https://wiki.ubuntu.com/{{latest_release_name}}/ReleaseNotes">Read the full release notes</a></p>
+        <p class="twelve-col"><a class="external" href="https://wiki.ubuntu.com/{{lts_release_no_point}}/ReleaseNotes">Read the full release notes</a></p>
```
Comment (anthonydillon) [inline templates/desktop/developers.html]: This link should be `https://wiki.ubuntu.com/XenialXerus/ReleaseNotes`.

File: templates/desktop/developers.html [hunk 2]
```diff
@@ -70,7 +70,26 @@
...
+            suite of apps, including a browser and a terminal. To access it, just select a ‘Unity 8’
+            session on the Greeter login screen.</p>
+        <div class="combined-list whats-new-graphic">
+            <ul class="list-ticks six-col">
+                <li>Unity 8 desktop interface (Ubuntu developer preview)</li>
+                <li>Selective Unity 8 apps: Ubuntu Browser, System Settings, Terminal</li>
+                <li class="last-item">Install and run snaps from the Terminal</li>
+            </ul>
+        </div>
+        <p class="twelve-col"><a class="external" href="https://wiki.ubuntu.com/{{latest_release_full}}/ReleaseNotes">Read the full release notes</a></p>
```
Comment (anthonydillon) [inline templates/desktop/developers.html]: Link should be https://wiki.ubuntu.com/YakketyYak/ReleaseNotes

**Comment by barrymcgee** [comment]
Good catch.. now amended.

**Comment by anthonydillon** [comment]
Thanks :+1:

**Pull request merged**